### PR TITLE
MDCB 2.5.1 Release Notes [DX-1285]

### PR DESCRIPTION
**Pull request by n2linux**
## **User description**
Release notes for MDCB 2.5.1

### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link
https://deploy-preview-4496--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-mdcb/release-notes/version-2.5/
<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

___

## **Type**
enhancement, documentation

___

## **Description**
- Added comprehensive release notes for MDCB version 2.5.1 in the documentation.
- Detailed third-party dependencies and compatibility, ensuring users are aware of the tested and compatible versions.
- Provided clear upgrade instructions from previous versions to 2.5.1.
- Included download links for the Docker image of MDCB version 2.5.1.
- Documented specific bug fixes related to health check port, API cache clearing, and PostgreSQL compatibility issues.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-2.5.md</strong><dd><code>Add Release Notes for MDCB Version 2.5.1</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.5.md
<li>Added detailed release notes for MDCB version 2.5.1.<br> <li> Included information on third-party dependencies, upgrade <br>instructions, and download links.<br> <li> Documented bug fixes and provided detailed explanations for each.<br> <li> No breaking changes or deprecations in this release.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4496/files#diff-7b39137d58210cd9bc2105b14da6d8327159ba1daa3dedff47964e050e6378a1">+63/-0</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Comment by dcs3spp** [comment]
@n2linux PR LGTM, will require peer review from @caroltyk before approval

**Review by dcs3spp — APPROVED** [review]
PR LGTM, @caroltyk could you peer review?

File: tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.5.md [hunk 1]
```diff
@@ -13,6 +13,69 @@ Licensed Protected Product
...
+   
+  When MDCB was configured with TLS enabled, traffic was served over HTTPS on the listen port that was configured. However, the healthcheck endpoint was exposed on the standard HTTPS port of 443 and TYK_MDCB_HEALTHCHECKPORT was not being respected.
+ </details>
+ </li>
+
+ <li>
+ <details>
+ <summary>Fixed a bug where clearing the API cache from the Tyk Dashboard UI failed to invalidate the cache in distributed data plane gateways</summary>
+
+  When clearing the API cache from the Tyk Dashboard UI, the cache in distributed data plane gateways was not being invalidated.
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.5.md]: Please add a note that this fix requires upgrading Gateway to 5.3.1 too.

**Review by carolwkc — CHANGES_REQUESTED** [review]

File: tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.5.md (hunk 1 above)
Comment (n2linux) [reply tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.5.md]: @caroltyk @dcs3spp Added the note.

Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-enterprise-mdcb/release-notes/version-2.5.md]: @caroltyk could you approve if ok and then I will merge and release to 5.3 docs

**Review by carolwkc — APPROVED** [review]
Thanks! LGTM

**Pull request merged**